src/js/modules/articles/actions.js

```diff
@@ -1,11 +1,55 @@
 import axios from 'axios';
 import * as t from './actionTypes';
+import STUY_SPEC_API from '../../constants';
 
-const apiUrl = 'http://api.icndb.com/jokes/random';
-
+//TODO: need a way to call the fetch function

```

Comment:
I generally call in the componentWillMount, but that's not great. I'll get back to you on this